build.gradle

```diff
@@ -94,16 +94,16 @@ dependencies {
 	implementation 'io.gsonfire:gson-fire:1.8.5'
 
 	// Use JUnit test framework
-	testImplementation 'junit:junit:4.13.2'
+	testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.2'
+	testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.2'
 
 	// https://github.com/cdimascio/java-dotenv
 	testImplementation 'io.github.cdimascio:java-dotenv:5.2.2'
 
 	// http://wiremock.org
-	testImplementation "com.github.tomakehurst:wiremock:2.27.2"
+	testImplementation "com.github.tomakehurst:wiremock-jre8:2.35.0"
```

Comment:
Maybe we could use wiremock-jre8-standalone instead?
See https://search.maven.org/search?q=g:com.github.tomakehurst